addon/-private/system/store/finders.js

```diff
@@ -52,9 +52,8 @@ export function _find(adapter, store, modelClass, id, internalModel, options) {
   }, `DS: Extract payload of '${modelName}'`);
 }
 
-export function _findMany(adapter, store, modelName, ids, internalModels) {
+export function _findMany(adapter, store, modelName, ids, snapshots) {
   if (DEBUG) { incrementRequestCount(); }
-  let snapshots = A(internalModels).invoke('createSnapshot');
   let modelClass = store.modelFor(modelName); // `adapter.findMany` gets the modelClass still
   let promise = adapter.findMany(store, modelClass, ids, snapshots);
```

Comment:
`snapshots` is a native array here, but was converted to an `EmberArray` in the now deleted [line 57](https://github.com/emberjs/data/pull/5054/files#diff-65c0e9ac4f030e1eff959291b8c5032aL57).

```js
let snapshots = A(internalModels).invoke('createSnapshot');
```

I should probably change this line from

```js
let promise = adapter.findMany(store, modelClass, ids, snapshots);
```

to

```js
let promise = adapter.findMany(store, modelClass, ids, A(snapshots));
```

Reply:
Apparently I should have [run the whole test suite](https://travis-ci.org/emberjs/data/jobs/369972161#L572-L591). Haha.

Seems like the `build-url-mixin` relies on `snapshots` being an `EmberArray`.

Reply:
Actually it was just the test, that relied on it. I've gone ahead and remove that `EmberArray` wrapper and fixed the test.